packaging/vagrant/Vagrantfile

```diff
@@ -26,7 +26,10 @@
 Vagrant.require_version ">= 2.1.3"
 Vagrant.configure("2") do |c|
 
-  c.vm.box = "centos/7"  # https://app.vagrantup.com/centos/boxes/7
+  c.vm.box = "centos/7"         # https://app.vagrantup.com/centos/boxes/7
+  c.vm.box_version = "~>1809"   # updates: https://blog.centos.org/?s=vagrant
+  c.vm.box_check_update = true  # warn if base box out of date
```

Comment:
This only produces a warning? Not a re-provision?

Reply:
Yes. Could be the behavior has changed or our prior understanding was wrong.

Reply:
Considering it was me, probably the latter ;) . We can merge unless anyone else need review.